### ISSUE TITLE
creatininuria

### ISSUE THREAD

**Issue by jpgourdine** [obophenotype/human-phenotype-ontology#4856]
**Preferred term label:**

**Synonyms**
Increased of creatinine in the urine

**Definition (free text, please give PubMed ID)**

An increase of creatinine levels in the urine (PMID: 23726637)

**Parent term (use hpo.jax.org/app)**

Abnormality of urine homeostasis HP:0003110

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

0000-0001-9969-8610

**Comment by pnrobinson** [comment]
Is this actually an abnormal phenotype? Urine contains creatinine normally and the concentration depends on many things.

**Comment by nicolevasilevsky** [comment]
maybe not - should we mark it unable to annotate?

**Comment by jpgourdine** [comment]
Yes, urinary creatinine can increase because of multiple factors (drugs, meat consumption but also men who suffer from diabetes)  DOI: https://doi.org/10.2215/CJN.09770913
<img width="631" alt="Screen Shot 2019-05-30 at 12 56 54 PM" src="https://user-images.githubusercontent.com/16328677/58660606-af79fe80-82da-11e9-8e20-0a8c1f0fca90.png"> @pnrobinson

**Comment by pnrobinson** [comment]
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3817914/

**Comment by nicolevasilevsky** [comment]
@pnrobinson do you think we should add this term to HPO or mark it unable to annotate in LOINC2HPO?

**Comment by pnrobinson** [comment]
@jpgourdine @nicolevasilevsky This is perhaps true but in the vast majority of cases, urinary creatinine is ued to normalize the concentration of other substances (e.g., https://www.kidney.org/atoz/content/understanding-your-lab-values). Therefore, in the vast majority of LOINC tests, there will be no relevance of the actual creatinine value. I do not think that we will get valid results if we add this to the LOINC algorithm at this time, without at least doing a detailed study of the values.
